app/assets/javascripts/miq_tree.js

```diff
@@ -394,21 +394,21 @@ function miqSquashToggle(treeName) {
 
 function miqTreeEventSafeEval(func) {
   var whitelist = [
-    'miqOnCheckCUFilters',
-    'miqOnCheckGenealogy',
+    'miqOnCheckCUFilters', // Settings -> Region -> CU Collection tab
+    'miqOnCheckGenealogy', // Compute -> Ingrastructure -> VMs -> Select one vm and click on genealogy
     'miqOnCheckGeneric',
-    'miqOnClickMenuRoles',
-    'miqOnCheckProtect',
-    'miqOnCheckSections',
+    'miqOnClickMenuRoles', // Settings -> Access Controll -> Roles/Edit Roles // Seems like not doing anything
+    'miqOnCheckProtect', // Compute -> Infra -> VMs -> Select VM -> Policy button -> edit
+    'miqOnCheckSections', // Compute -> Infra -> VMs -> Compare VM's
     'miqOnCheckUserFilters',
-    'miqOnClickAutomate',
-    'miqOnClickAutomateCatalog',
-    'miqOnClickDiagnostics',
-    'miqOnClickGeneric',
-    'miqOnClickHostNet',
-    'miqOnClickSnapshots',
-    'miqOnClickUtilization',
-    'miqOnCheckTenantTree',
+    'miqOnClickAutomate', // Atuomate -> Atuomate // Maybe not used
+    'miqOnClickAutomateCatalog', // Maybe not used
```

Comment:
Used for entrypoint selection when creating a new catalog item and probably also for copying an automate method.

Reply:
You go to `Services -> Catalogs -> Catalog Items` and try to add a new item. If you select `Generic`, there will be three entry point fields available. Each of them is able to open this tree.

Reply:
AFAIK @ZitaNemeckova is actually converting stuff in the area, maybe she can share some insights.

Reply:
@skateman I think every type except Ansible Playbook can open it.